scripts/validate-artifacts.ps1

```diff
@@ -137,5 +183,6 @@
     throw "Unable to get a reponse from the web app. Please make sure that the VM and web application are running and try again."
```

Comment:
The script attempts to access a web application on port 8080. Ensure that the web application is running and accessible at the specified address to avoid errors.